virttest/env_process.py

```diff
@@ -526,7 +527,11 @@ def validate_memory_resource():
         if (params.get("setup_ksm") == "yes" and
                 params.get("ksm_run", "1") == "1"):
             magnification = 1.2
-        free_mem = "%s KB" % memory.freememtotal()
+        # Test whether is MemAvailable already introduced
+        if 'MemAvailable' in genio.read_file('/proc/meminfo'):
+            free_mem = "%s KB" % memory.read_from_meminfo('MemAvailable')
```

Comment:
@will-Do concernign import of 'genio' module check https://github.com/avocado-framework/avocado-vt/pull/948#discussion_r107670586 where @lmr  proposed it.
The second thing .. try - except... I would rather avoid this. It is not good habit to use exceptions for flow control (http://schlitt.info/opensource/blog/0724_python_good_bad_evil_03_flow_control_exceptions.html)
But I can avoid double opening `/proc/meminfo` file by using regular expressions:
```
meminfo = genio.read_file('/proc/meminfo')
if 'MemAvailable' in meminfo:
    memfree = re.findall(r'MemAvailable:\s+(\d+)', meminfo)
else:
    memfree = re.findall(r'MemFree:\s+(\d+)', meminfo)
```
so what is your opinion @will-Do ?